test/fixtures/test-page-object.html

```diff
@@ -41,12 +42,14 @@
 
     <p style="height: 800px">Strut to make the page scroll</p>
     <script>
+      var liveAnnouncementUpdateIndex = 0;
       var button = document.getElementById('button');
       button.addEventListener('click', function () {
         var message = document.createElement('span');
         message.id = 'click-message';
         message.textContent = 'Clicked!';
         document.body.insertBefore(message, button.nextElementSibling);
+        document.getElementById('live-announcement').textContent = `update ${++liveAnnouncementUpdateIndex}`;
```

Comment:
Can we have separate buttons for different scenarios to avoid unexpected side effects?